pootle/apps/pootle_language/views.py

```diff
@@ -108,3 +112,26 @@ def language_admin(request, language):
     }
     return admin_permissions(request, language.directory,
                              'languages/admin/permissions.html', ctx)
+
+
+@get_path_obj
+@permission_required('administrate')
+def language_characters_admin(request, language):
+    if request.method == 'POST':
+        form = LanguageSpecialCharsForm(request.POST, instance=language)
```

Comment:
Finally managed to get it working using CBV, but the changeset is larger: https://github.com/translate/pootle/compare/translate:master...unho:language-characters-cbv Since I find simpler to use a function view I will keep it as the proposed fix.